roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DiskAvailability(OpenShiftCheck):
+    """Check that recommended disk space is available."""
+
+    name = "disk_availability"
+    tags = ["preflight"]
+
+    recommended_diskspace = {
+        "nodes": 15*10**9,
+        "masters": 40*10**9,
+        "etcd": 20*10**9,
+    }
+
+    minimum_docker_space_percent = 5.0
+    supported_docker_dev_driver = "devicemapper"
```

Comment:
Yeah, I was thinking of removing the check for the storage driver from this PR altogether, and just handling it as part of the `docker_storage_check` PR https://github.com/openshift/openshift-ansible/pull/3787/files#diff-c3d0f0c6a0d94ad7d6c5a62da695118dR17